src/pages/App/duplication-localStorage.js

```diff
@@ -0,0 +1,11 @@
+export const duplicationCheckLocalStorage = (arrOfDuplication) => {
+  if (arrOfDuplication.length > 0) {
+    let errorMsgArr = [];
+    for (let i = 0; i < arrOfDuplication.length; i++) {
+      errorMsgArr.push(
+        `This house is already in your list: ${arrOfDuplication[i].address}`
+      );
+    }
+    return errorMsgArr.join();
+  }
+};
```

Comment:
```suggestion
export const duplicationCheckLocalStorage = (arrOfDuplication) => {
  if (arrOfDuplication.length === 0) {
    return;
  }

  const errorMsgArr = [];
  arrOfDuplication.forEach((item) => {
    errorMsgArr.push(`This house is already in your list: ${item.address}`);
  });
  return errorMsgArr.join();
};
```
nitpick: because we only want to perform this check when the array has elements, we can use a guard clause at the start of the method and use an early return. This helps reduce a layer of nesting in our code which is usually easier to read.

Additionally we can make use of the `forEach` function which is a more modern approach to iterating over arrays.

Reply:
Your suggestion code looks much easier to read. I should always remember to do early return. I changed it to forEach too.

Reply:
Early returns are a nice pattern to follow for sure!